presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Standard.java

```diff
@@ -99,10 +125,47 @@ public static DockerContainer createPrestoContainer(DockerFiles dockerFiles, Pat
     {
         return new DockerContainer(dockerImageName)
                 .withFileSystemBind(dockerFiles.getDockerFilesHostPath(), "/docker/presto-product-tests", READ_ONLY)
+                .withFileSystemBind(dockerFiles.getDockerFilesHostPath("conf/presto/etc/jvm.config"), CONTAINER_PRESTO_JVM_CONFIG, READ_ONLY)
                 .withFileSystemBind(pathResolver.resolvePlaceholders(serverPackage).toString(), "/docker/presto-server.tar.gz", READ_ONLY)
                 .withCommand("/docker/presto-product-tests/run-presto.sh")
                 .withStartupCheckStrategy(new IsRunningStartupCheckStrategy())
                 .waitingFor(Wait.forLogMessage(".*======== SERVER STARTED ========.*", 1))
                 .withStartupTimeout(Duration.ofMinutes(5));
     }
+
+    private void enableDebugger(CreateContainerCmd createContainerCmd)
```

Comment:
Doesn't launcher support to pass JVM properties? It would be much easier that this.

Reply:
doesn't, i have already put a PR

Reply:
Please add a comment about this